### ISSUE TITLE
Chart Padding is Uneven

### ISSUE THREAD

**Issue by rasmusvan** [perses/perses#2126]
### What did you do?

Embedded dashboard into my application.  Hid y-axis data

### What did you expect to see?

Even padding on the right and left of the chart within the panel. Ideally in proportion to the size of the chart width (getting smaller if the chart gets leaner)

### What did you see instead? Under which circumstances?

The padding on the left is always bigger than on the right. This is true even when there is data on the y-axis, but especially obvious, when the y-axis is hidden. The smaller the panel, the more visually unappealing it is, as the padding size remains the same.

### System information

Darwin 23.5.0 arm64

### Perses version

0.46.0

### Perses configuration file

```yaml
Here is the minimal reproduction repository for this problem.

Branch "chart-padding-problem".

https://github.com/rasmusvan/perses-embedded-dashboard-mrr.git
```

**Comment by Gladorme** [comment]
Hello, could you please provide a screenshot and the panel type, please

**Comment by rasmusvan** [comment]
Here is a screenshot from the minimal reproduction repository that I provided:
<img width="1919" alt="image" src="https://github.com/user-attachments/assets/58dab89f-9440-4708-945e-0c4b8d0261c9">
As you can see, the left padding is always bigger than the right.

I'm guessing that by panel type you mean the chart kind, which is TimeSeriesChart

**Comment by Gladorme** [comment]
Fixed in PR #2143
